pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2428,3 +2446,11 @@ func isInUsbGroup(aa types.AssignableAdapters, ib types.IoBundle) bool {
 	}
 	return false
 }
+
+func getRoofFsPath(rootPath string) string {
+	return path.Join(rootPath, containerRootfsPath)
+}
+
+func getCtrdSnapshotID(rootPath string) string {
```

Comment:
The new paths will be /persist/vault/volume/<volume UUID>#<generation counter>
VolumeID is not a snapshotID I think.

Reply:
If you boot a device in alpha with an app instance I think you can already see that created snapshot/volume (it isn't used yet - Rishab is working on the zedmanager+volumemgr changes to use it). But it gets created.

Reply:
I veririfed that it works fine with the current/future pathnames.